### PR TITLE
doc(readme): Fix Typos and add additional line breaks

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
Spotted a few typos while using this library. Also added some line breaks between paragraphs and headings, and correctly capitalised product names, etc.

### CODE REVIEW

**Comment by vandie** [comment]
Hey Paul, thanks for the PR. Will review the changes shortly.

File: readme.md [hunk 1]
```diff
@@ -1,94 +1,115 @@
...
 
 You can now grab the type of mention that you need for any given page.
+
 ```typescript
 import webMentionHandler from 'path/to/your/webMentionHandler/file.ts';
 
-const likes = webMentionHandler.getMentionsForPage('/blog/example-post', 'likes');
-const comments = webMentionHandler.getMentionsForPage('/blog/example-post', 'commments');
+const likes = webMentionHandler.getMentionsForPage('/blog/example-post', 'like');
+const replies = webMentionHandler.getMentionsForPage('/blog/example-post', 'reply');
```
Comment (vandie) [inline readme.md:92]: I was going to say this was incorrect, but looking at the code I use on https://mikevdv.dev. Seems that I updated this query at some point and didn't update the docs

**Review by vandie — APPROVED** [review]
@paulrobertlloyd Changes wise, I'm happy. However, your PR doesn't follow the conventional commit rules. Can you please correct this?

https://www.conventionalcommits.org/en/v1.0.0/

**Comment by paulrobertlloyd** [comment]
Opps, missed that. Have renamed the commit.

**Pull request merged**